kotlin-styled-next/src/main/kotlin/styled/GlobalStyles.kt

```diff
@@ -89,6 +106,30 @@ object GlobalStyles {
         }
     }
 
+    internal fun removeStyles(styledCss: StyledCss) {
+        if (sheet !is CSSOMSheet) return
+        val sheet = sheet as CSSOMSheet
+        val info = styledClasses[styledCss] ?: throw IllegalStateException("Trying to remove non-existent css")
+        info.usedBy--
+        if (info.usedBy == 0) {
+            scheduledToDelete.add(styledCss)
+        }
+        sheet.requestClean { clean(sheet) }
+    }
+
+    private fun clean(sheet: CSSOMSheet) {
+        val removalGroups = scheduledToDelete.mapNotNull { css ->
+            styledClasses[css]?.let {
```

Comment:
Problem here that sometimes styledClasses[css] is null, can't find out why. Works with just skipping nulls

Reply:
Do not remove items during iteration, rewrite it using two passes.

Reply:
Worked with `HashSet` instead of `MutableList`, thanks